websites/D/Daily Mail/newsPage.ts

```diff
@@ -0,0 +1,34 @@
+import { findPage } from "./findPage";
+import { Assets } from "./presence";
+
+function handleArticle(presenceData: PresenceData): void {
+	const aBody = document.querySelector("#js-article-text"),
+		titleEl = aBody.querySelector("h1");
+	presenceData.buttons = [{ label: "Read Article", url: window.location.href }];
+
+	if (titleEl.querySelector(".is-exclusive")) {
+		presenceData.details = "Reading Article";
+		presenceData.state = titleEl.textContent.replace("EXCLUSIVE", "");
+		presenceData.smallImageKey = Assets.exclIco;
+		presenceData.smallImageText = "Exclusive!";
+	} else {
+		presenceData.details = "Reading Article";
+		presenceData.state = titleEl.textContent;
+		presenceData.smallImageKey = "";
```

Comment:
```suggestion
		delete presenceData.smallImageKey;
```

Reply:
huh, I didn't realize you could just `delete` it